docs/docker.rst

```diff
@@ -19,6 +19,13 @@ To compile ``blinky1.hex`` targeting an AVR microcontroller such as the Arduino:
 
     docker run --rm -v $(pwd):/src tinygo/tinygo build -o /src/blinky1.hex -size=short -target=arduino examples/blinky1
 
+For projects that have remote dependencies outside of the standard library and go code within your own project, you will need to map your entire GOROOT into the docker image in order for those dependencies to be found.  
+
+    docker run  -v $(PWD):/mysrc -v $GOPATH:/gohost -e "GOPATH=$GOPATH:/gohost" tinygo/tinygo build -o /mysrc/wasmout.wasm -target wasm /mysrc/wasm-main.go
+
+NOTE: 
+- At this time, tinygo does not resolve dependencies from the /vendor/ folder within your project. 
```

Comment:
You can make this a real note with some extra markup: https://sublime-and-sphinx-guide.readthedocs.io/en/latest/notes_warnings.html